clientmanager/k8s/starter.py

```diff
@@ -101,11 +102,13 @@ def make_k8s_job_desc(
         "limits": {
             "cpu": str(n_cores),
             # TODO: give a bit more just in case?
-            "memory": memory.upper().replace("B", ""),  # 4Gb -> 4G
+            "memory": worker_memory.upper().replace("B", ""),  # 4Gb -> 4G
+            "ephemeral-storage": worker_disk.upper().replace("B", ""),  # 4Gb -> 4G
```

Comment:
Considering this need to remove "B", you could do this globally, since condor will also accept the unit without the "B" at the end.